web-client/src/app/state/session.service.ts

```diff
@@ -144,6 +144,15 @@ export class SessionService {
     };
     const result: UpdateOtpPhoneNumberResult =
       await this.enclaveService.updateOtpPhoneNumber(request);
+
+    if ('Updated' in result) {
+      this.sessionStore.update((state) => {
+        if (state.wallet) {
```

Comment:
no need to do this because you can only update the session when you are logged and you can't have a session without a wallet